inst/init/standing.ppmx

```diff
@@ -510,3 +520,30 @@ PMX_VPC:
     'y': DV
   is.smooth: no
   smooth_with_bloq: no
+
+## SAEM plots
+    
+SAEM_CONVERGENCE:
```

Comment:
While this can be use for `saem_convergence` some people also adjust and look at `focei`; perhaps the name can be generalized;  In `saem` this is a convergence plot, but in other methods it is a parameter history plot.

Reply:
Sounds good to me.